themes/apiarybasestyles/source/css/buttons.styl

```diff
@@ -10,3 +10,10 @@
     background-color #264653
     &:hover
       background-color #2A9D8F
+
+.loadingIndicator
+  $loadingIndicator()
+  &--big
+    $loadingIndicator(size: 12px)
+  &--blue
+    $loadingIndicator(color: $primaryColor)
```

Comment:
@kuba-kubula Do you care to do a PoC/PR with this? Can't imagine how do you mean it.

And as of parsing time, I think at the size of CSS we operate and other issues we solve it's a minor problem, but I did some googling and it seems like that's not true http://csswizardry.com/2016/02/mixins-better-for-performance/